watertap/tools/parameter_sweep/parameter_sweep_writer.py

```diff
@@ -222,27 +183,48 @@ def _write_outputs(self, output_dict, txt_options="metadata"):
         # We will also create a companion txt file by default which contains
         # the metadata of the h5 file in a user readable format.
         txt_fname = self.config["h5_results_file_name"] + ".txt"
-        if txt_options == "metadata":
-            my_dict = copy.deepcopy(output_dict)
-            for key, value in my_dict.items():
-                for subkey, subvalue in value.items():
-                    subvalue.pop("value")
-        elif txt_options == "keys":
-            my_dict = {}
-            for key, value in output_dict.items():
-                if key != "solve_successful":
-                    my_dict[key] = list(value.keys())
-        else:
-            my_dict = output_dict
 
-        with open(txt_fname, "w") as log_file:
-            pprint.pprint(my_dict, log_file)
+        if self.config.h5_parent_group_name is not None and os.path.isfile(txt_fname):
+            return
+        else:
```

Comment:
Strictly speaking, we do not need the `else` clause here

Reply:
Got rid of the empty if statement.